<launch>
	<!-- <node name="pole_extractor_node_upper" pkg="pole_structure_mapper" type="pole_extractor_node" respawn="false" output="screen">
		<param name="gripper_tag_frame_id" type="string" value="U"/>
		<param name="filter_box_size" type="double" value="1.5"/>
		<param name="min_diameter" type="double" value="0.10"/>
		<param name="max_diameter" type="double" value="0.20"/>
	</node> -->

	<node name="pole_extractor_node_lower" pkg="pole_structure_mapper" type="pole_extractor_node" respawn="false" output="screen">
		<param name="gripper_tag_frame_id" type="string" value="L"/>
		<param name="filter_box_size" type="double" value="1.5"/>
		<param name="min_diameter" type="double" value="0.10"/>
		<param name="max_diameter" type="double" value="0.20"/>
	</node>

</launch>
